<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileUploadUI</class>
 <widget class="QDialog" name="FileUploadUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileUploadUI</string>
  </property>
  <widget class="QPushButton" name="buttonChoose">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUploadFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload File</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFileName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatusMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
